db/db_with_timestamp_basic_test.cc

```diff
@@ -324,19 +368,16 @@ TEST_F(DBBasicTestWithTimestamp, UpdateFullHistoryTsLow) {
   }
   ASSERT_OK(Flush());
 
-  for (int i = 0; i < 10; i++) {
+  // TODO return a non-ok for read ts < current_ts_low and test it.
```

Comment:
Question: wouldn't the original test code work with one tiny change, namely that we would return `NotFound` for `i < current_ts_low - 1` instead of `i < current_ts_low` ? (Same below)

Reply:
It will work now, but I am considering returning another status code, e.g. "read timestamp too low", to indicate the actual error. For now, I can follow this suggestion.

Reply:
Actually, sometimes ts zeroing will occur, thus reading even with ts=0 can return result.